core/chain-configs/src/client_config.rs

```diff
@@ -264,17 +264,20 @@ pub struct ReshardingConfig {
 
     /// The delay between attempts to start resharding while waiting for the
     /// state snapshot to become available.
+    /// UNUSED in ReshardingV3.
```

Comment:
Nice! 

Is there any sort of "deprecated" tag that we could use, that will show a warning if those fields are set?

Reply:
I was thinking we could remove these fields entirely? Do you foresee any issue?

Reply:
Ah yeah that's even better. I thought you're keeping them for the resharding v2 replayability but as long as we're past that phase let's get rid of it.